packages/babel-generator/src/source-map.js

```diff
@@ -34,7 +34,7 @@ export default class SourceMap {
    * values to insert a mapping to nothing.
    */
 
-  mark(generatedLine: number, generatedColumn: number, line: number, column: number, filename: ?string) {
+  mark(generatedLine: number, generatedColumn: number, line: number, column: number, identifierName: ?string, filename: ?string) {
```

Comment:
Getting max-len error here (do we still want this on), I'd be ok with removing as well)